bigframes/dataframe.py

```diff
@@ -1473,6 +1473,48 @@ def cov(self, *, numeric_only: bool = False) -> DataFrame:
 
         return result
 
+    def corrwith(
+        self,
+        other: typing.Union[DataFrame, bigframes.series.Series],
+        *,
+        numeric_only: bool = False,
+    ):
+        other_frame = other if isinstance(other, DataFrame) else other.to_frame()
+        if not numeric_only:
```

Comment:
Let's re-write this if-else block as

if numeric_only:
   ...
else
   ...

See go/tott/685